.gitignore

```diff
@@ -1,7 +1,6 @@
-/Cargo.lock
```

Comment:
Is it intentional to remove `Cargo.lock` from git ignore?

Reply:
Yes, I added `Cargo.lock` to prevent the dependencies from updating (similar we use in `radixdlt-scrypto`)